myapp/listenPort.js

```diff
@@ -0,0 +1,10 @@
+const request = require('request');
+const app = require('./app.js');
+
+const listenPort = function () { 
```

Comment:
could be directly exported from this point no need to declare a const variable:
module.exports.startServer = () => {
.....
}